CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.14)
 set(CMAKE_CXX_STANDARD 17)
 
+
```

Comment:
bitte entfernen

Reply:
I merged that from master but I can still remove it

Reply:
and it breaks the compilation, as expected

Reply:
ok removed

Reply:
commit 8223effb6c4b6f06076d2056d9244e4af7d498e8